### ISSUE TITLE
Comment.

### ISSUE THREAD

**Issue by Eugenexjay** [MyasnikovIA/Termux-Java-8#1]
Its is working 100% installation to termux ?

**Comment by MyasnikovIA** [comment]
Yes, everything works great. Moreover, I wrote a Web server in Java and after installing OpenJDK I made a server based on Android TV
![photo_2023-12-21_12-23-25](https://github.com/MyasnikovIA/Termux-Java-8/assets/12913376/4b7b1088-ea70-453b-905f-e114b9befe80)

**Comment by MyasnikovIA** [comment]
Project WebServerLinte for Termux 
https://github.com/MyasnikovIA/WebServerLite

**Comment by MyasnikovIA** [comment]
Video of the startup process on Android TV

https://www.youtube.com/watch?v=y9txGx5kDxI&ab_channel=%D0%98%D0%B2%D0%B0%D0%BD%D0%9C%D1%8F%D1%81%D0%BD%D0%B8%D0%BA%D0%BE%D0%B2

**Comment by MyasnikovIA** [comment]
Python lovers can install this interpreter in Termux (pkg install python). At the time of writing this post, Python version 3.11 was available
![Install_python](https://github.com/MyasnikovIA/Termux-Java-8/assets/12913376/f6d7e663-c70c-4dd5-aa59-113e9ec82631)

**Comment by Eugenexjay** [comment]
Bro, is that java openjdk8 "version 1.8.0_152" running on termux ?

Like this !!!
![Screenshot_20231209-185758.png](https://github.com/MyasnikovIA/Termux-Java-8/assets/120492432/0715a528-0a6e-4c96-9ddd-86297bb8da38)

???

**Comment by MyasnikovIA** [comment]
Стабильная версия OpenJDK 1.7. Пытался найти версию 1.8, но столкнулся с проблемой с кодировкой русских букв. Вот почему я откатил разработку приложений до старой версии JAVA. Очень крутая сборка виртуальной машины, которую я использую при написании приложения для Windows, — это сборка из OpenLogic Java.
----
Stable version of OpenJDK 1.7. I tried to find version 1.8, but ran into a problem with the encoding of Russian letters. That's why I rolled back my application development to an old version of JAVA. A very cool virtual machine build that I use when writing a Windows application is the build from OpenLogic Java.

**Comment by MyasnikovIA** [comment]
![java_version](https://github.com/MyasnikovIA/Termux-Java-8/assets/12913376/b9f56baf-8d25-4c11-9c01-ec75982cc17f)
![java_version_win](https://github.com/MyasnikovIA/Termux-Java-8/assets/12913376/9b1a11ff-7a2f-4b5c-9e4b-0cae6a8a1916)

**Comment by MyasnikovIA** [comment]
Вообще то ты сам можешь скомпилировать Termux под свои нужды. Мне понадобилось сделать из Android TV маленький сервер. вот из за этого я погрузился в этот проект. Но из коробки в нем не был реализован автозапуск, вот и пришлось его дописывать. Проект открытый и при необходимости ты можешь его доработать на  свое усмотрение(Android Studio).   
-----------------
In fact, you can compile Termux yourself to suit your needs. I needed to make a small server from Android TV. That is why I plunged into this project. But out of the box it did not have autorun implemented, so I had to add it. The project is open and, if necessary, you can modify it at your own discretion (Android Studio).

**Comment by MyasnikovIA** [comment]
Tested installation of OpenJDK for Termux:
 https://raw.githubusercontent.com/MyasnikovIA/Termux_boot_ssh/main/termux-shared/src/main/res/raw/install_open_jdk_11_sh
 https://raw.githubusercontent.com/MyasnikovIA/Termux_boot_ssh/main/termux-shared/src/main/res/raw/install_open_jdk_8_sh
 https://raw.githubusercontent.com/MyasnikovIA/Termux_boot_ssh/main/termux-shared/src/main/res/raw/install_open_jdk_9_sh

------
Проверенная установка OpenJDK для Termux:
